src/Nox.Generator/Application/Dto/EntityUpdateDto.template.cs

```diff
@@ -37,9 +37,9 @@ public partial class {{className}}
     /// {{entity.Name}} {{relationship.Description}} {{relationship.Relationship}} {{relationship.EntityPlural}}
     /// </summary>
     {{- if relationship.Relationship == "ZeroOrMany" || relationship.Relationship == "OneOrMany"}}
-    public virtual List<{{relationship.Entity}}UpdateDto> {{relationship.EntityPlural}} { get; set; } = new();
+    public virtual List<{{relationship.Entity}}Dto> {{relationship.EntityPlural}} { get; set; } = new();
     {{- else}}
-     public virtual {{relationship.Entity}}UpdateDto{{- if relationship.Relationship == "ZeroOrOne"}}?{{end}} {{relationship.Entity}} { get; set; } = null!;
+    public virtual {{relationship.Entity}}Dto{{- if relationship.Relationship == "ZeroOrOne"}}?{{end}} {{relationship.Entity}} { get; set; } = null!;
```

Comment:
Why this change is needed?, the owned in the update should also be an updateDto? for example we can not set autogenerated ids?

Reply:
Thanks, let then talk about this next monday. Can you please update with the latest from main, so it's easier to understand the changes,

Reply:
I've updated the final PR that includes all the changes for this ticket - https://github.com/NoxOrg/Nox.Generator/pull/673